doc/content/getting-started/troubleshooting/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Troubleshooting"
+description: ""
+---
+
+This section provides help for common issues and frequently asked questions you may have when getting started with {{% tts %}}. If you encounter a bug, please [file an issue](https://github.com/TheThingsNetwork/lorawan-stack/issues/new/choose). If you have questions not covered here, please ask in [{{% tts %}} Forum](https://www.thethingsnetwork.org/forum/c/network-and-routing/v3).
+
+<!--more-->
+
+## My gateway won't connect. What do I do?
+
+- Double check your gateway settings in {{% tts %}}. Ensure your Gateway EUI is correct
+- Double check the configuration settings on your gateway. Is the network address the same address you use to connect to {{% tts %}}? Are your ports correct?
+- If using CUPS, double check your API Keys and certificates
+
+## My device won't join. What do I do?
+
+- Double check your Device EUI, Join EUI, LoRaWAN Version, AppKey, and NwkKey
+- Double check frequency plan settings in your end device and gateways (they must be matching)
+- Double check your network connection. If there is a slow connection from the server to the gateway, the join accept message may be sent too late (this can happen when a gateway uses 3G as a backhaul)
+- Check for an overused App Nonce
```

Comment:
Check gateway and application events too - if you see no traffic in the application events the gateway events may give some insight.

Reply:
Should we also add a `{{% tts %}} no longer receives traffic from my device. What do I do ?` ? I'd point to the gateway logs in this section. If no traffic can be seen in the application/device, but the gateway logs keep saying 'not found' the device is not getting matched probably.

Reply:
How does this happen after a device has joined?

Reply:
An ABP device may reset its counters, due to a power cycle, but since there is no join mechanism the NS cannot know this happened. The result is that the NS says 'well you say you are device 000000001234 but that device should have FCnt 1337 not 1 like you' and then fails to match this device. 

In these cases the user has to manually reset the FCnt (right @rvolosatovs ?)

Reply:
If an ABP device did reset frame counters and `mac_settings.resets_f_cnt` is set to `true`, NS will recognize that automatically, also in 1.1 there are MAC commands that indicate that - manual frame counter resets should not be necessary often

Reply:
"JoinNonce (or AppNonce)" for consistency